src/LibChorus/Properties/launchSettings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "profiles": {
+    "LibChorus": {
+      "commandName": "Executable",
+      "executablePath": "C:\\fwrepo\\flexbridge\\output\\Debug\\net461\\FLExBridge.exe",
+      "commandLineArgs": "-u developer -p c:\\fwrepo\\fw\\DistFiles\\Projects -v obtain -projDir c:\\fwrepo\\fw\\DistFiles\\Projects -fwAppsDir C:\\fwrepo\\fw\\Output\\Debug -fwmodel 7000072 -liftmodel 0.13_ldml3 -locale en -pipeID SendReceivec:\\fwrepo\\fw\\DistFiles\\Projectsobtain"
```

Comment:
I'm not sure if it makes sense to check in this file since it contains hard-coded paths. 

I can understand that it might be useful to have this setup available. I'd prefer if we would put it in the readme, or rename this file to something like `launchSettings.json.sample` - that way it doesn't imply that it should work out of the box. And we should put `launchSettings.json` in the `.gitignore` file so that it doesn't get accidentally committed.

Reply:
done